bazel/test_runners/qemu_with_kernel/run_qemu.sh

```diff
@@ -76,6 +76,11 @@ flags+=(-append "console=ttyS0 root=/dev/sda")
 # Disable graphics mode.
 flags+=(-nographic)
 
+if ! virt-host-validate qemu -q; then
```

Comment:
The docker container doesn't have this command. Also I think we probably want to just fix the error checking on the `qemu-system` command, such that any qemu error is returned as an exit code. The reason it doesn't return the error currently is because we transform the error code (because we assume the error code is coming from the isa exit device which is twos-complemented), so an error code of 1 from qemu becomes 0.

Reply:
ah, I didn't consider the docker use case. I was aware that we transformed the error code, which is why I initially thought handling this before the qemu command may be more convenient.

I'll take a look into that error code transformation and see how this could be handled.

Reply:
We can also add the command to the docker container, but its unclear to me that if `qemu-system` were to fail, `virt-host-validate qemu` would be guaranteed to fail

Reply:
I think @zasgar also had a fix in the works for this

Reply:
Sounds good. I'll wait to see if this is already addressed in any unmerged changes.